sindresorhus/refined-github
# Fully restore `set-default-repositories-type-to-sources`

Thanks for contributing! 🍄

1. LINKED ISSUES: Fixes #3733 

2. TEST URLS:
    1. Click your "user dropdown" (top-right corner of every page)
    2. Click "Your repositories"

Should we also take the opportunity to drop the selectors used for the old layout?
